app/packages/state/src/hooks/useSchemaSettings.ts

```diff
@@ -30,19 +33,22 @@ const skipFiled = (path: string, ftype: string) => {
   return (
     ftype === DICT_FIELD ||
```

Comment:
[`fieldsReducer`](https://github.com/voxel51/fiftyone/blob/adc271617adb47e062b41736610d222513126a94/app/packages/state/src/recoil/sidebar.ts#L147) and/or [`UNSUPPORTED_FILTER_TYPES`](https://github.com/voxel51/fiftyone/blob/adc271617adb47e062b41736610d222513126a94/app/packages/utilities/src/index.ts#L318) could be leveraged here